perun-core/src/test/java/cz/metacentrum/perun/core/api/AuthzResolverIntegrationTest.java

```diff
@@ -1345,6 +1346,61 @@ public void getVoRichAdminsWithProperRights() throws Exception {
 		assertTrue(richAdmins.containsAll(Arrays.asList(testUser, testUser2)));
 	}
 
+	@Test
+	public void getVoAdminsWithProperRights() throws Exception {
+		System.out.println(CLASS_NAME + "getVoAdminsWithProperRights");
+
+		final Vo testVo = perun.getVosManager().createVo(sess, new Vo(0,"testvo1","testvo1"));
+		final Group testGroup = perun.getGroupsManager().createGroup(sess, testVo, new Group("testGroup", "testg"));
+		final Member testMember = createSomeMember(testVo);
+		final User testUser = perun.getUsersManagerBl().getUserByMember(sess, testMember);
+		final Member testMember2 = createSomeMember(testVo);
+		final User testUser2 = perun.getUsersManagerBl().getUserByMember(sess, testMember);
+
+		perun.getGroupsManager().addMember(sess, testGroup, testMember2);
+
+		PerunPrincipal mockedPerunPrincipal = mock(PerunPrincipal.class, RETURNS_DEEP_STUBS);
+		when(mockedPerunPrincipal.isAuthzInitialized()).thenReturn(true);
+		when(mockedPerunPrincipal.getRoles()).thenReturn(new AuthzRoles(Role.VOADMIN, testVo));
+		when(mockedPerunPrincipal.getRolesUpdatedAt()).thenReturn(System.currentTimeMillis());
+		when(mockedPerunPrincipal.getActor()).thenReturn("test");
+		PerunSession testSession = new PerunSessionImpl(sess.getPerun(), mockedPerunPrincipal, sess.getPerunClient());
+
+		AuthzResolver.setRole(testSession, testUser, testVo, Role.VOADMIN);
+		AuthzResolver.setRole(testSession, testGroup, testVo, Role.VOADMIN);
+		List<User> admins = AuthzResolver.getAdmins(testSession, testVo, Role.VOADMIN, false);
+
+		assertEquals(2, admins.size());
+		assertTrue(admins.containsAll(Arrays.asList(testUser, testUser2)));
+	}
+
+	@Test
+	public void getDirectVoAdminsWithProperRights() throws Exception {
+		System.out.println(CLASS_NAME + "getVoAdminsWithProperRights");
+
```

Comment:
I think this should be fixed to reflect the actual name of this test
```suggestion
	@Test
	public void getDirectVoAdminsWithProperRights() throws Exception {
		System.out.println(CLASS_NAME + "getDirectVoAdminsWithProperRights");

```